Activities/Cryptography/UiPath.Cryptography.Activities/DecryptFile.cs

```diff
@@ -69,6 +70,20 @@ public partial class DecryptFile : CodeActivity
         [LocalizedDescription(nameof(Resources.Activity_DecryptFile_Property_ContinueOnError_Description))]
         public InArgument<bool> ContinueOnError { get; set; }
 
+        [Browsable(false)]
+        [DefaultValue(null)]
+        [LocalizedCategory(nameof(Resources.Input))]
+        [LocalizedDisplayName(nameof(Resources.Activity_DecryptFile_Property_InputFile_Name))]
+        [LocalizedDescription(nameof(Resources.Activity_DecryptFile_Property_InputFile_Description))]
+        public InArgument<IResource> InputFile { get; set; }
```

Comment:
We need to either document what takes priority between this and the file path as string, or throw an exception if both are set to prevent confusion.

Please take this up with product. Also this impacts other activities I think.

Reply:
As discussed with Product, an exception is raised when both are filled in. Text message of exception is validated with Product as well.